.github/workflows/tracks.yml

```diff
@@ -0,0 +1,39 @@
+name: Tracks
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  race:
+    name: Run `${{ matrix.track }}` on stack ${{ matrix.stack-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        stack-version: ["8.3.2"]
+        track:
+          - noaa
+      fail-fast: false
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```

Comment:
Rally itself supports python `3.8`, `3.9`, and `3.10`. Should tracks be tested across at least the min and max supported versions as well? How does that get matrix-ified?

Reply:
With the latest commit I submitted, in trying to make the track/challenge/test-mode flexibility, I made difficult adding new dimensions to the matrix.

Github Actions are not very smart in this regards, either you go fully-generated combinations or you go fully-manual.